### PR TITLE
Added Accessibility category

### PR DESCRIPTION
**Pull request by simonsan**
I've added the `accessibility` category to cover crates that implement features for the convenience of disabled people.

I wanted to start a project that translates given colour codes to colours colorblind people can distinguish more easily, but I haven't found a category to fit it. What else belongs in this category could be crates that interoperate with peoples screenreaders, new screenreader implementations or other assistive technology.

### CODE REVIEW

File: src/boot/categories.toml [hunk 1]
```diff
@@ -26,6 +26,12 @@
 #   `@`, `:`, or `.`. They should be all lowercase.
 #
 
+[accessibility]
+name = "Accessibility"
+description = """
+Crates that implement features for the convenience of disabled people. \
```
Comment (jtgeibel) [inline src/boot/categories.toml:32]:
```suggestion
Tools for making your creations usable by as many people as possible. \
```

**Review by jtgeibel — CHANGES_REQUESTED** [review]
Thanks for the PR @simonsan. I drew some inspiration from the [web accessibility docs](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/What_is_accessibility#So_what_is_accessibility) and propose an alternate description.

**Comment by simonsan** [comment]
> 
> 
> Thanks for the PR @simonsan. I drew some inspiration from the [web accessibility docs](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/What_is_accessibility#So_what_is_accessibility) and propose an alternate description.

Thanks for the input. ;-) Changed it.

**Comment by jtgeibel** [comment]
Thanks @simonsan!

@bors r+

**Pull request merged**